docs/graphics/utilities/hook_focus_visible.svg

```diff
@@ -0,0 +1 @@
52-7.006 1 1 0 0 1 1.402-.183Zm86.386 0a1 1 0 0 1 1.402.184 30.953 30.953 0 0 1 4.052 7.006 1 1 0 1 1-1.847.766 28.98 28.98 0 0 0-3.791-6.553 1 1 0 0 1 .184-1.402Zm6.424 14.147a1 1 0 0 1 1.121.862 31.257 31.257 0 0 1 0 8.096 1 1 0 1 1-1.983-.26A29.23 29.23 0 0 0 205 90c0-1.285-.083-2.55-.245-3.789a1 1 0 0 1 .862-1.12Zm-99.234 0a1 1 0 0 1 .862 1.121A29.23 29.23 0 0 0 107 90c0 1.285.083 2.55.245 3.789a1 1 0 1 1-1.983.259 31.257 31.257 0 0 1 0-8.096 1 1 0 0 1 1.121-.862Zm1.51 15.47a.999.999 0 0 1 1.307.541 28.98 28.98 0 0 0 3.791 6.553 1 1 0 0 1-1.586 1.219 30.974 30.974 0 0 1-4.052-7.006 1 1 0 0 1 .54-1.307Zm96.214 0a1 1 0 0 1 .54 1.307 30.974 30.974 0 0 1-4.052 7.006 1 1 0 1 1-1.586-1.219 28.98 28.98 0 0 0 3.791-6.553.999.999 0 0 1 1.307-.541Zm-87.163 12.633a1 1 0 0 1 1.402-.184 28.98 28.98 0 0 0 6.553 3.791.999.999 0 1 1-.766 1.847 30.974 30.974 0 0 1-7.006-4.052 1 1 0 0 1-.183-1.402Zm78.112 0a1 1 0 0 1-.183 1.402 30.974 30.974 0 0 1-7.006 4.052 1 1 0 1 1-.766-1.847 28.98 28.98 0 0 0 6.553-3.791 1 1 0 0 1 1.402.184Zm-63.966 6.424a1 1 0 0 1 1.121-.862A29.23 29.23 0 0 0 136 119h3.333a1 1 0 1 1 0 2H136c-1.371 0-2.722-.089-4.048-.262a1 1 0 0 1-.862-1.121Zm49.82 0a1 1 0 0 1-.862 1.121A31.277 31.277 0 0 1 176 121h-3.333a1 1 0 1 1 0-2H176a29.23 29.23 0 0 0 3.789-.245 1 1 0 0 1 1.121.862ZM145 120a1 1 0 0 1 1-1h6.667a1 1 0 0 1 0 2H146a1 1 0 0 1-1-1Zm13.333 0a1 1 0 0 1 1-1H166a1 1 0 0 1 0 2h-6.667a1 1 0 0 1-1-1Z" clip-rule="evenodd"/><path fill="#4A4A4A" d="M113 90c0-13.255 10.745-24 24-24h39c13.255 0 24 10.745 24 24s-10.745 24-24 24h-39c-13.255 0-24-10.745-24-24Z"/><path fill="#fff" d="M135.069 94.827c2.409 0 3.903-1.193 3.903-3.099 0-1.395-1.088-2.468-2.582-2.58v-.136c1.158-.165 2.042-1.163 2.042-2.31 0-1.674-1.306-2.702-3.496-2.702H130v10.827h5.069Zm-2.707-9.146h1.971c1.127 0 1.776.525 1.776 1.41 0 .9-.696 1.411-1.971 1.411h-1.776v-2.821Zm0 7.458v-3.144h2.05c1.392 0 2.151.54 2.151 1.553 0 1.036-.736 1.591-2.097 1.591h-2.104Zm16.325-6.573h-2.277v4.772c0 1.133-.61 1.831-1.744 1.831-1.048 0-1.588-.593-1.588-1.763v-4.84h-2.276v5.38c0 1.921 1.158 3.054 3.019 3.054 1.307 0 2.112-.54 2.519-1.5h.141v1.327h2.206v-8.26Zm2.933-1.973v2.048h-1.346V88.3h1.346v4.3c0 1.606.829 2.258 2.925 2.258.438 0 .861-.045 1.142-.097v-1.613a5.601 5.601 0 0 1-.673.037c-.774 0-1.118-.345-1.118-1.088V88.3h1.791v-1.658h-1.791v-2.048h-2.276Zm6.695 0v2.048h-1.345V88.3h1.345v4.3c0 1.606.829 2.258 2.926 2.258.438 0 .86-.045 1.142-.097v-1.613a5.614 5.614 0 0 1-.673.037c-.774 0-1.119-.345-1.119-1.088V88.3h1.792v-1.658h-1.792v-2.048h-2.276ZM168.046 95c2.651 0 4.247-1.606 4.247-4.307 0-2.679-1.619-4.307-4.247-4.307s-4.248 1.636-4.248 4.307c0 2.694 1.596 4.307 4.248 4.307Zm0-1.726c-1.22 0-1.909-.945-1.909-2.581 0-1.62.696-2.574 1.909-2.574 1.204 0 1.908.953 1.908 2.574 0 1.636-.696 2.581-1.908 2.581Zm6.109 1.553h2.276v-4.78c0-1.117.649-1.838 1.705-1.838 1.079 0 1.588.608 1.588 1.771v4.847H182v-5.342c0-1.973-1.032-3.099-2.949-3.099-1.275 0-2.151.578-2.55 1.508h-.141v-1.328h-2.205v8.261Z"/></svg>
```

Comment:
Ditto for these files — is there a reason for using snake_case in the file names?

Reply:
Changed from underscore to dash to follow pattern

<img width="276" alt="Screen Shot 2022-03-24 at 6 02 34 PM" src="https://user-images.githubusercontent.com/10593890/160017712-b0e51377-0df2-4ee0-ad02-b04d4f17fefc.png">

Reply:
DONE